src/abstracts/SablierV2Lockup.sol

```diff
@@ -199,6 +200,11 @@ abstract contract SablierV2Lockup is
         streamedAmount = _streamedAmountOf(streamId);
     }
 
+    /// @inheritdoc ERC721
+    function supportsInterface(bytes4 interfaceId) public view override(ERC721, IERC165) returns (bool) {
```

Comment:
Components with fewer inherited components should be inherited first.

i.e. `IERC165` should come before `ERC721`.

Reply:
`SablierV2Lockup` does not inherit `IERC165`/`ERC165`. `ERC721` inherits them. So, I added `ERC721` before `IERC165` to make it follow the inheritance tree (Lockup inherits ERC721 inherits ERC165). Do you still think `IERC165` should come before `ERC721`?